### ISSUE TITLE
Questions about nnUNet automatic configuration

### ISSUE THREAD

**Issue by sdwiodhi** [Yanfeng-Zhou/nnWNet#3]
Dear authors, hello. Thank you for publishing an excellent paper and for also open-sourcing your code. I have a small question: Did your experiments use all the default parameter configurations of nnUNet, such as the number of stages for the encoder and decoder, and the number of channels in each stage, etc. (I only found this in the paper: "Other configurations (e.g., learning rate, loss function, patch size, and optimizer) are the default configurations in the nnUNet framework." I found fixed channel numbers in the code: "layer_channel=[16, 32, 64, 128, 256]," but it seems that parameters were not read from the plan.json file.)
If you could take the time to clarify my doubts, I would be greatly appreciated. I am eager to know whether you used the stages configured in nnUNet. Thank you again!

**Comment by Yanfeng-Zhou** [comment]
All configurations of the UNet architecture follow the original nnUNet framework, and we did not modify it. For layer_channel=[16, 32, 64, 128, 256], this is a parameter we designed specifically for WNet and has nothing to do with the original nnUNet architecture.

Please see nnunetv2.training.nnUNetTrainer.nnUNetTrainer for details of the nnUNet trainer.

**Comment by sdwiodhi** [comment]
Thank you for your prompt response. In my understanding, in nnU-Net, the number of channels in convolutional layers is automatically configured based on the dataset. For example, nnunet automatically configures the number of channels for both encoders and decoders, including the number of layers in each stage. In your code, I saw different-sized models like L, S, M, etc., which are configured with different numbers of channels but have the same number of stages. Can I understand it this way: for different datasets, the number of channels in your network architecture is fixed and does not dynamically adjust based on changes in the dataset? (However, the models used, such as WNet3D, WNet3D_M, WNet3D_S, can be manually adjusted)?
I look forward to your reply again. Thank you once again for taking the time to clarify my doubts amidst your busy schedule.

> All configurations of the UNet architecture follow the original nnUNet framework, and we did not modify it. For layer_channel=[16, 32, 64, 128, 256], this is a parameter we designed specifically for WNet and has nothing to do with the original nnUNet architecture.所有 UNet 架构的配置均遵循原始 nnUNet 框架，我们并未对其进行修改。对于 layer_channel=[16, 32, 64, 128, 256]，这是我们专门为 WNet 设计的参数，与原始 nnUNet 架构无关。
> 
> Please see nnunetv2.training.nnUNetTrainer.nnUNetTrainer for details of the nnUNet trainer.请参阅 nnunetv2.training.nnUNetTrainer.nnUNetTrainer 了解 nnUNet 训练器的详细信息。

**Comment by Yanfeng-Zhou** [comment]
Yes, the code for automatic architecture configuration is too complicated and we did not implement it. But note that for the eight datasets in the paper, the architecture configuration of nnUNet is the same, all [32, 64, 128, 256, 512, 512, 512]. So you can assume that for most datasets of appropriate size, nnUNet follows this configuration.

**Comment by sdwiodhi** [comment]
Thank you for your enthusiastic reply. Wishing you a happy life and smooth research. Thanks again!

> Yes, the code for automatic architecture configuration is too complicated and we did not implement it. But note that for the eight datasets in the paper, the architecture configuration of nnUNet is the same, all [32, 64, 128, 256, 512, 512, 512]. So you can assume that for most datasets of appropriate size, nnUNet follows this configuration.是的，自动架构配置的代码过于复杂，我们没有实现它。但请注意，在论文中的八个数据集上，nnUNet 的架构配置是相同的，都是[32, 64, 128, 256, 512, 512, 512]。所以你可以假设对于大多数适当大小的数据集，nnUNet 遵循这个配置。
